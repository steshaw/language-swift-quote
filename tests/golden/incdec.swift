var a = 0
let b = ++a
// a and b are now both equal to 1
let c = a++
// a is now equal to 2, but c has been set to the pre-increment value o
