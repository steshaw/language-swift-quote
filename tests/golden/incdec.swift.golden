var a = 0
let b = ++a
let c = a++
